test/request/accepts.js

```diff
@@ -8,8 +8,14 @@ describe('ctx.accepts(types)', function(){
     describe('when Accept is populated', function(){
       it('should return all accepted types', function(){
         const ctx = context();
-        ctx.req.headers.accept = 'application/*;q=0.2, image/jpeg;q=0.8, text/html, text/plain';
-        ctx.accepts().should.eql(['text/html', 'text/plain', 'image/jpeg', 'application/*']);
+        ctx.req.headers.accept =
```

Comment:
I can remove that limit and rebase if that works better, it'll definitely be a bit less weird.
@tejasmanohar ok, that's easy fix too.